src/wp-includes/class-wp-term-query.php

```diff
@@ -457,6 +457,14 @@ public function get_terms() {
 				"tt.taxonomy IN ('" . implode( "', '", array_map( 'esc_sql', $taxonomies ) ) . "')";
 		}
 
+		if ( '' === $args['exclude'] ) {
+			$args['exclude'] = array();
+		}
+
+		if ( '' === $args['include'] ) {
+			$args['include'] = array();
+		}
```

Comment:
Could `exclude` be any empty or even a string that is only spaces? If yes, then I'd suggest the following:

```suggestion
		if ( empty( $args['exclude'] ) ) {
			$args['exclude'] = array();
		}

		if ( empty( $args['include'] ) ) {
			$args['include'] = array();
		}
```

Reply:
Empty would also be trigger is 0 and '0', is this what we want here?

Reply:
> Empty would also be trigger is 0 and '0', is this what we want here?

Yes, that's good.

Currently `get_terms( [ 'include' => 0 ] );` runs the default query & returns `uncatagorized` on a fresh install. As there is never a term with the ID `0`, an emtpy array for `exclude` makes sense too.